pkg/kubernetes/details.js

```diff
@@ -3,12 +3,130 @@ define([
     "base1/cockpit",
     "base1/angular",
     "base1/term",
+    "kubernetes/client",
     "kubernetes/app"
-], function($, cockpit, angular, Terminal) {
+], function($, cockpit, angular, Terminal, kubernetes) {
     'use strict';
 
     var _ = cockpit.gettext;
 
+    var k8client =  kubernetes.k8client();
+    var adjust_btn = $('#adjust-entity-button'); 
+    var adjust_entity_dlg = $('#adjust-entity-dialog');
+    var delete_btn = $('#delete-entity-button'); 
+    var delete_entity_dlg = $('#delete-entity-dialog');
+    var delete_pod_dlg = $('#delete-pod-dialog');
+    var delete_pod_btn = $('#delete-pod-button');
+
+    delete_entity_dlg.on('show.bs.modal', function(e) {
+        var name = $(e.relatedTarget).attr("data-name");
```

Comment:
I would suggest just setting a 'data-key' attribute here. And using that key with client.objects, rather than using three different attributes to convey the same information.